src/memefactory/shared/graphql_schema.cljs

```diff
@@ -69,6 +69,7 @@
     params(db: String!, keys: [String!]): [Parameter]
     overallStats : OverallStats
     config: Config
+    events: [Event]
```

Comment:
I'm not seeing who is populating and consuming the collection, what is the purpose of it?

Reply:
healthcheck endpoint - we can query for the last seen events, get the event count before resyncing etc